Python-Function-MAX.md

```diff
@@ -0,0 +1,49 @@
+# Python max(x)
+
+`max()` is a built-in function in Python 3. It returns the largest item in an iterable or the largest of two or more arguments.
+
+## Arguments
+This function takes two or more numbers or any kind of iterable as an argument. While giving an iterable as an argument we must make sure that all the elements in the iterable are of the same type. This means that we cannot pass a list which has both string and integer values stored in it.
```

Comment:
Yes.
It can take both int and floats and it can even compare them together.
It works for strings too. It checks for lexicographical order.
eg: max('abc', 'abb', 'abc') # returns 'abc'